ui/apps/platform/src/Components/PatternFly/FormMessage.tsx

```diff
@@ -4,7 +4,6 @@ import { Alert, AlertVariant } from '@patternfly/react-core';
 export type FormResponseMessage = {
     message: string;
     isError: boolean;
-    data?: any;
```

Comment:
Curious about this change. Do we not actually use data, or are we just not typing it, so that it can be free-form (basically an implicit typing of any) ?

Reply:
Good eyes!

This component does not use the `data` property.

Saif, can you confirm that `responseData` is the key of the optional property, and here are its uses:
* https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/ApiTokenIntegrationForm/ApiTokenFormMessageAlert.tsx#L15-L32
* https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/ClusterInitBundleIntegrationForm/ClusterInitBundleFormMessageAlert.tsx#L69-L75